lib/logic.rb

```diff
@@ -0,0 +1,35 @@
+# rubocop:disable Layout/LineLength, Metrics/CyclomaticComplexity, Metrics/PerceivedComplexity, Metrics/AbcSize
+class WinCondition
+  attr_reader :final_arr
+
+  def initialize
+    @final_arr = []
+  end
+
+  def win(arr, sim, player)
+    if (arr[0] == sim and arr[1] == sim and arr[2] == sim) || (arr[3] == sim and arr[4] == sim and arr[5] == sim) || (arr[6] == sim and arr[7] == sim and arr[8] == sim)
+      puts "\n !!!!!!!!!!!!#{player} wins !!!!!!!!!!! \n".upcase
+      @final_arr.concat(arr)
+      arr.clear
```

Comment:
- [ ] No `puts/print` and `gets.chomp` outside `main.rb` file. ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/ruby/OOP/milestone3/only_main_print.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59565018/))